docs/source/conf.py

```diff
@@ -102,14 +141,22 @@
 autodoc_inherit_docstrings = False
 
 
+# -- katex javascript in header
+#
+#def setup(app):
```

Comment:
Do you need this to render katex? Should we keep it?

Reply:
the comment should stay, for reference, but the macros can go.

Reply:
lets remove the commented function then.